packages/flutter_tools/test/general.shard/android/android_install_test.dart

```diff
@@ -93,13 +94,13 @@ void main() {
     expect(await androidDevice.installApp(androidApk), false);
   });
 
-  testWithoutContext('Can install app on API level 16 or greater', () async {
+  testWithoutContext('Can install app on API level minSdk or greater', () async {
```

Comment:
Nit: maybe include the minSdk value here in the test name?

Something like:

`Can install app on API level ${gradle_utils.minSdkVersion} or greater`

or:

`Can install app on API level minSdk (${gradle_utils.minSdkVersion}) or greater`